index.js

```diff
@@ -1,4 +1,29 @@
 'use strict';
 const pMap = require('p-map');
 
-module.exports = (count, mapper, opts) => pMap(Array(count).fill(), (el, i) => mapper(i), opts);
+const symbolEnd = Symbol('pTimes.end');
+
+module.exports = (count, mapper, opts) => {
+	const results = [];
+	let times = 0;
+
+	for (let i = 0; i < count; i++) {
+		results.push(mapper(i));
+	}
```

Comment:
This doesn't preserve the `concurrency` option. I think the easiest solution would be to add `.end` support to `p-map` too, so it can be just reused here.